[PL-43422] note to use vanity URL instead of app.harness (#6616)
harness/developer-hub: docs/platform/authentication/single-sign-on-saml.md

	| **Valid post logout redirect URIs** | `https://app.harness.io/ng/account/<YOUR ACCOUNT ID>/main-dashboard`                                                                       |
	| **Master SAML Processing URL** | `https://app.harness.io/gateway/api/users/saml-login?accountId=<YOUR ACCOUNT ID>`                                                               |

:::info note

> If the account uses vanity URL, then use the same in SAML setup. eg: https://<yourvanityurl>/gateway/api/users/saml-login?accountId=<YOUR ACCOUNT ID>

Reviewer: ```suggestion
If the account uses a vanity URL, then use the vanity URL in your SAML setup. For example, `https://<yourvanityurl>/gateway/api/users/saml-login?accountId=<YOUR ACCOUNT ID>`.
```

Reviewer: This update will fix the failing PR build.

Reviewer: Recommend avoiding i.e. and e.g. in developer docs.

Author: Changes done, can you please approve, will merge it post approval
